frontend/canvas-chrome-ext/.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea
```

Comment:
we already have a gitignore, i think you can just add these to the one in the project root directory

Reply:
Ah, yes. I'll fix that. I was working with the frontend code only so intelliJ would load the JS packages needed since the whole project has two languages and confused intelliJ